hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/OzoneClientConfig.java

```diff
@@ -124,6 +125,15 @@
       tags = ConfigTag.CLIENT)
   private boolean checksumVerify = true;
 
+  @Config(key = "checksum.combine.mode",
+      defaultValue = "MD5MD5CRC",
+      description = "The combined checksum type [MD5MD5CRC / COMPOSITE_CRC] "
+          + "determines which algorithm would be used to compute checksum for "
+          + "file checksum. Default checksum type is MD5MD5CRC.",
+      tags = ConfigTag.CLIENT)
+  private String checksumCombineMode =
+      Options.ChecksumCombineMode.MD5MD5CRC.name();
```

Comment:
Please make the default value to COMPOSITE_CRC. I think it makes more meaningful in Ozone because the ask to implement #getFileChecksum() API came from replication work.